projects/packages/paypal-payments/src/block/product-placeholder.scss

```diff
@@ -0,0 +1,107 @@
+@import '@automattic/jetpack-base-styles/gutenberg-base-styles';
+@import '../../shared/styles/jetpack-variables';
```

Comment:
I believe you should be able to replace this to remove the dependency.

Reply:
Thanks for catching this. For this one, I simply replaced the variable call with the value in the `product-placeholder.scss` file and removed the dependency. Let me know if that's what you had in mind. See [15e21df](https://github.com/Automattic/jetpack/pull/43413/commits/15e21df90d306b760c163277e0e8fa14779b7e43).